test/e2e/framework/autoscaling/autoscaling_utils.go

```diff
@@ -59,11 +66,17 @@ const (
 	rcIsNil                         = "ERROR: replicationController = nil"
 	deploymentIsNil                 = "ERROR: deployment = nil"
 	rsIsNil                         = "ERROR: replicaset = nil"
+	crdIsNil                        = "ERROR: CRD = nil"
 	invalidKind                     = "ERROR: invalid workload kind for resource consumer"
 	customMetricName                = "QPS"
 	serviceInitializationTimeout    = 2 * time.Minute
 	serviceInitializationInterval   = 15 * time.Second
 	megabytes                       = 1024 * 1024
+	crdVersion                      = "v1"
+	crdKind                         = "TestCRD"
+	crdGroup                        = "example.com"
```

Comment:
can you make the group name more explicitly related to autoscaling e2e (e.g. `autoscalinge2e.example.com`)? that's especially important if we're going to install this and not clean it up

Reply:
Sure, good idea. Done.